packages/shared/lib/clients/orchestrator.ts

```diff
@@ -77,86 +76,113 @@ export class Orchestrator {
+                'action.name': actionName,
+                'connection.id': connection.id,
+                'connection.connection_id': connection.connection_id,
+                'connection.provider_config_key': connection.provider_config_key,
+                'connection.environment_id': connection.environment_id
+            };
+            if (isGloballyEnabled || isEnvEnabled) {
+                const span = tracer.startSpan('execute.action', {
+                    tags: spanTags,
+                    ...(activeSpan ? { childOf: activeSpan } : {})
+                });
                 try {
                     const groupKey: string = 'action';
                     const executionId = `${groupKey}:environment:${connection.environment_id}:connection:${connection.id}:action:${actionName}:at:${new Date().toISOString()}:${uuid()}`;
                     const parsedInput = JSON.parse(JSON.stringify(input));
                     const args = {
-                        name: actionName,
+                        actionName,
                         connection: {
                             id: connection.id!,
+                            connection_id: connection.connection_id,
                             provider_config_key: connection.provider_config_key,
                             environment_id: connection.environment_id
                         },
                         activityLogId,
                         input: parsedInput
                     };
-                    // Execute dry-mode: no await for now
-                    void this.client
-                        .executeAction({
-                            name: executionId,
-                            groupKey,
-                            args,
-                            timeoutSettingsInSecs: {
-                                createdToStarted: 5,
-                                startedToCompleted: 5,
-                                heartbeat: 10
-                            }
-                        })
-                        .then(
-                            (res) => {
-                                if (res.isErr()) {
-                                    logger.error(`Error: Execution '${executionId}' failed: ${stringifyError(res.error)}`);
-                                } else {
-                                    logger.info(`Execution '${executionId}' executed successfully with result: ${JSON.stringify(res.value)}`);
-                                }
-                            },
-                            (error) => {
-                                logger.error(`Error: Action '${executionId}' failed: ${stringifyError(error)}`);
-                            }
-                        );
+                    const actionResult = await this.client.executeAction({
+                        name: executionId,
+                        groupKey,
+                        args,
+                        timeoutSettingsInSecs: {
```

Comment:
since there is a method dedicated, shouldn't this object be just the default value?

Reply:
you are right. I had custom values for the dry run but this is not necessary anymore. Thanks